move/move-ibc/sources/ibc.move

```diff
@@ -340,6 +334,10 @@ module IBC::Core {
         client_state: vector<u8>,
         consensus_state: vector<u8>,
     ) acquires IBCStore, SignerRef {
+        // NOTE(aeryz): At this point, we don't need to have a routing mechanism because it will introduce
+        // additional gas cost. We should only enforce the use of `cometbls` for the `client_type`
+        assert!(string::bytes(&client_type) == &b"cometbls", E_UNKNOWN_CLIENT_TYPE);
```

Comment:
we should make an issue to track this probably, and benchmark how much gas this actually saves

Reply:
#2919